README.md

```diff
@@ -1,7 +1,7 @@
 # Mario Kart 8 Deluxe Random Build Generator
 
-This discord bot creates you a random build containing a set of 1 character, 1 body, 1 tire and 1 glider and also displays the stats of this build.  
-It's also possible to create multiple builds at the same time up to 12. 
+This discord bot creates a random build containing a set of 1 character, 1 body, 1 tire and 1 glider and also displays the stats of this build.  
+It's also possible to create multiple builds at the same time, up to 12. 
```

Comment:
After reading this over and over again I think rephrasing makes more sense.

Should be, if you agree to it as well, reads way better IMO:
*"It's also possible to create up to 12 builds at the same time."*

Reply:
It has been changed 👍🏾